docs/api/taxons.rst

```diff
@@ -931,3 +931,63 @@ Exemplary Response
 .. code-block:: text
 
     STATUS: 204 No Content
+
+Set position of product in a Taxon
+----------------------------------
+
+The products in Sylius can by grouped by taxon, therefore for every product there is a relation between the product and the assigned taxon.
+What is more, every product can have a specific position in the taxon to which it belongs. To put products in a specific order
+you will need to call the ``/api/v1/taxons/taxon_code/products`` endpoint wih the ``PUT`` method.
+
+Definition
+^^^^^^^^^^
+.. code-block:: text
+
+    PUT /api/v1/taxons/{taxonCode}/products
+
++---------------+----------------+-----------------------------------------------------------------+
+| Parameter     | Parameter type | Description                                                     |
++===============+================+=================================================================+
+| Authorization | header         | Token received during authentication                            |
++---------------+----------------+-----------------------------------------------------------------+
+| code          | url attribute  | Code of the taxon in which the order of product will be changed |
++---------------+----------------+-----------------------------------------------------------------+
+
+Example
+^^^^^^^
+
+To change the order of products with codes ``7f260b98-aad8-3932-b628-1ac135d47767`` and ``c7127357-db24-3931-adb8-e80fc44e1507`` in taxon with code ``womens_t_shirts`` use the below method:
+
+.. code-block:: bash
+
+    $ curl http://demo.sylius.org/api/v1/taxons/womens_t_shirts/products \
+        -H "Authorization: Bearer SampleToken" \
+        -H "Accept: application/json" \
+        -X PUT
+        --data '
+            {
+                "positionsData": [
```

Comment:
1. It should not be camel cased
2. `positionsData` is not a good name. `product_positions`?